### ISSUE TITLE
Washington has less confirmed cases today (18th of March) than yesterday.

### ISSUE THREAD

**Issue by lewinmadrid** [CSSEGISandData/COVID-19#1060]
Washington data is strange today,
The one death has disappeared (Perhaps resurrected) and the number of confirmed case has reduced from 1076 to 1014.

**Comment by bioinfoacademy** [comment]
its 1187 for 18th march. 
CNN reported 1187 in their last update yesterday.
CDC has this number today as of 2.02 EST.

**Comment by adrimos** [comment]
Yes, there are many inconsistencies in the WA state data. I reported the same issue. Here is the latest from doh:
https://www.doh.wa.gov/Emergencies/Coronavirus
1187 seems the correct number of confirmed cases. It was also the number reported by the Seattle Times today for 3/18/20. 66 deaths for 3/18/20

**Comment by engineerchange** [comment]
Note #1011, #1012, #1013 - it appears the data on the website somehow deviated from the data in GitHub.

**Comment by bioinfoacademy** [comment]
Not just washington, the numbers are off for california, alabama, arkansas, compared to CDC report for yesterday... i dont event want to check on others...

**Comment by bioinfoacademy** [comment]
> Note #1011, #1012, #1013 - it appears the data on the website somehow deviated from the data in GitHub.

FWIW, we will provide US state-by-state curated data, starting today, at https://github.com/bioinfoacademy/covid-19-us/blob/master/data/time_series_19-covid-Confirmed-us-current.csv
This data has total for every state starting from 1/22.
All future daily updates to this file will come from CDC resource.
The data will be updated once a day, with CDC's 4 pm cut-off.
